src/lib.rs

```diff
@@ -42,6 +42,7 @@ pub struct Analysis {
     pub refs: Vec<Ref>,
     pub macro_refs: Vec<MacroRef>,
     pub relations: Vec<Relation>,
+    pub borrows: Vec<Borrows>,
```

Comment:
Each `Borrows` object represents all of the borrows in a single function/closure. The vec represents the collection of borrows for all functions/closures in a crate. I could have also taken another approach and simply moved its internals (assignments, loans, moves) up, but this would have required more processing to be done within the compiler.  I took the current approach because it was simplest based on how the borrow checker structures its data and required the least transformations.

Naming things is not my strong suit, so I'm very willing to change this name to something clearer, or restructuring the structs if that makes more sense.